### PR TITLE
Community School Integration block.

### LINKED ISSUE #38: Add Community School Integration block
School_Data

### PR DESCRIPTION
**Pull request by bstopp**
Fix #38 

Test URLs:
- Before: https://main--hsf-commonmoves--hlxsites.hlx.live/drafts/stopp/communities/acton
- After: https://feat-community-schools--hsf-commonmoves--hlxsites.hlx.live/drafts/stopp/communities/acton

### CODE REVIEW

File: blocks/liveby-schools/liveby-schools.css [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+.liveby-schools .school-lists.show-all section .school-item:nth-of-type(n+5) {
+  display: flex;
+}
+
+.liveby-schools .school-lists section .school-item .school-details {
+  display: flex;
+  position: relative;
+  flex-direction: column;
+  flex: 1 1;
```
Comment (irenelagno) [inline blocks/liveby-schools/liveby-schools.css:79]: i think you can use  flex shorthand to include flex-basis as well. see https://www.w3schools.com/cssref/css3_pr_flex.php

File: blocks/liveby-schools/liveby-schools.js [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+  header.textContent = `${type} Schools`;
+  section.append(header);
+
+  sorted.forEach((s) => {
+    let title;
+    if (s.overviewLink) {
+      title = `<a href="${s.overviewLink}" target="_blank">${s.name}</a>`;
+    } else {
+      title = s.name;
+    }
```
Comment (irenelagno) [inline blocks/liveby-schools/liveby-schools.js:67]: you can simplify code by 
```
let title = s.name;
if (s.overviewLink) {
  title = `<a href="${s.overviewLink}" target="_blank">${title}</a>`;
```
}

**Comment by AmyJZhao** [comment]
There's an "Explore More Communities" button under the title. Is that going to be another custom block for later?
Screenshot 2023-04-05 at 3 55 21 PM

**Review by irenelagno — CHANGES_REQUESTED** [review]

**Comment by bstopp** [comment]
> There's an "Explore More Communities" button under the title that's missing from your page. Is that going to be another custom block to implement later?
> 
> Screenshot 2023-04-05 at 3 55 21 PM
> 
> 

Different block. The only thing in this PR is the list of school info.

**Comment by bstopp** [comment]
> Also one column layout for smaller screens vs 3 columns:

I intentionally made the 900px width have 3 columns instead of one. Trying to limit to our agreed breakpoints of 600/900/1200. If i don't do it at 900px, then it won't swap to 3col until 1200px.

**Comment by bstopp** [comment]
I added #45 to track the demographics issue, which isn't related to this PR, but a good catch.

**Review by irenelagno — APPROVED** [review]

**Pull request merged**